applications/FemToDemApplication/custom_processes/update_pressure_value_pfem_conditions_process.cpp

```diff
@@ -34,12 +34,11 @@ void UpdatePressureValuePfemConditionsProcess<TDim>::Execute()
         auto it_cond = it_cond_begin + i;
         auto& r_geometry = it_cond->GetGeometry();
         double average_pressure = 0.0;
-        unsigned int number_wet_nodes = 0;
 
         for (unsigned int i = 0; i < r_geometry.PointsNumber(); i++) {
```

Comment:
OMP¿?

Reply:
the loop over the conditions it's in omp :) line 32